### DISCUSSION TITLE
How do I change the index page of my profile?

### DISCUSSION THREAD

**Discussion by MortenSmidth** [community/community#18177] [Other Feature Feedback, Questions, & Ideas]
How do I change the index page of my profile?

**Comment by MathiasReker** [comment]
Hello

You need to create a repository of your GitHub name and add a `README.md` file to it.

**Comment by savanrajkotiya** [comment]
Hello, 
You will need to simply make a repo of your username & add `README.md` file in that.
 Also hereby I'm dropping some open-source repo links that will provide cool features to a make attractive `README.md`file.
 1. https://rahuldkjain.github.io/gh-profile-readme-generator
 2. https://gprm.itsvg.in/
